application/__init__.py

```diff
@@ -36,11 +36,13 @@ def handle_error(e):
     from application.controllers.timers import TimersResource
     from application.controllers.rests import RestsResource
     from application.controllers.exercises import ExercisesResource, RandExercises
+    from application.controllers.users import UsersResource
 
     api.add_resource(TimersResource, '/timers', '/timers/<int:timer_id>')
```

Comment:
Just a note, I assume most of the other endpoints will need to modified at some point too, since each user will have a timer, rest/focus intervals, and potentially exercise list